src/block/input.css

```diff
@@ -0,0 +1,31 @@
+.search-form__input {
+  height: 100%;
+  width: 100%;
+  padding: 0;
+
+  font-size: 16px;
+
+  border: none;
+}
+
+.search-form__input::placeholder {
+  color: inherit;
+}
+
+.search-form__input--small {
+  font-size: 14px;
+}
+
+.search-form__input:focus {
+  outline: none;
+  font-weight: 900;
+
+  background: linear-gradient(
+    var(--first-gradient-form-color),
+    var(--second-gradient-form-color)
+  );
```

Comment:
Ensure that the background gradient on focus enhances the usability and accessibility of the input field. If the gradient makes the text hard to read or the focus state less discernible, consider using a simpler background color change or a border to indicate focus.